### PR TITLE
Uyuni upgrade clients #12527 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [ ] Manager-4.2
- [ ] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I hear @juliogonzalez asking for a PDF :wink:

**Comment by keichwa** [comment]
[uyuni_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/5385623/uyuni_client-configuration_guide.pdf)

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
+[[client-upgrades-uyuni]]
+= Upgrade Uyuni Clients
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/client-upgrades-uyuni.adoc:2]: You are talking about openSUSE Leap here only. Specifically to upgrade to Leap 15.2.

Remember SLE can also be managed with Uyuni.

File: modules/client-configuration/pages/client-upgrades.adoc [hunk 2]
```diff
@@ -1,16 +1,21 @@
 [[client-upgrades]]
 = Client Upgrades
 
+ifeval::[{suma-content} == true]
 Clients use the versioning system of their underlying operating system, and require regular upgrades.
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/client-upgrades.adoc:5]: This is also valid for Uyuni. Uyuni can manage SLE.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
The procedure to upgrade Leap clients to Leap 15.2 on Uyuni seems correct.

However, it looks to me that we need a different structure. You are assuming "Uyuni Clients" == "openSUSE Leap", but that's not really correct.

First of all, Uyuni clients can also be RHEL, CentOS, Oracle, Ubuntu, Debian... and if a user sees "Upgrade clients", they could expect information about it.

But it can also manage SLE.

So maybe you need to rethink this and have:

- A single page to upgrade SLE clients, as it's the same procedure for Uyuni and SUSE Manager
- A page to upgrade openSUSE Leap clients on Uyuni
- A page to upgrade openSUSE Leap clients on SUSE Manager (AFAIK not possible yet because of SCC limitations, @mcalmer can say more)

And then link the stuff at the nav guide, and fix the `client-upgrades.acoc`

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc [hunk 3]
```diff
@@ -57,8 +59,11 @@ Key: 	  re-1-1008b9843d44dac77f0a856f207cc2c6
 . In the [guimenu]``Child Channels`` pane, select the {opensuse-version} child channels.
 . Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
 . Click menu:Software[Packages > Upgrade], and select all the packages to be updated on the client, and then apply the selection.
-. Re-register with the reactivation key using the [command]``rhnreg_ks`` command-line utility.
-  The system will be re-registered with the same id, history, and groups.
+  Click btn:[Upgrade Packages], and then btn:[Confirm].
```
Comment (Loquacity) [inline modules/client-configuration/pages/client-upgrades-uyuni.adoc:62]:
```suggestion
  Click btn:[Upgrade Packages], check the details, and click btn:[Confirm] to complete the update.
```

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/client-upgrades-uyuni.adoc:2]: Thanks, we now have https://github.com/SUSE/spacewalk/issues/12793 --I just created it ;)

File: modules/client-configuration/pages/client-upgrades.adoc (hunk 2 above)
Comment (keichwa) [reply modules/client-configuration/pages/client-upgrades.adoc:5]: ok, but sb must sort out the details ;)  With this PR I tried to solve an issue a user reported on the uyuni ML (#12527).  Of course, your concerns are very valid but we must track and solve them separately.

I just created https://github.com/SUSE/spacewalk/issues/12795.

**Comment by paususe** [comment]
@juliogonzalez
Upgrade of Leap clients on SUSE Manager will be enabled soon by SCC

**Comment by keichwa** [comment]
there is a N XREF ERROR but I think it will vanish once merged...

**Pull request merged**